### ISSUE TITLE
no way to return from the export page

### ISSUE THREAD

**Issue by zoltee** [phpmyadmin/phpmyadmin#12586]
### Steps to reproduce
1. run a query and click the export link at the bottom of the list
2. try to return to the previous page (result list)

### Expected behaviour
there should be a button/link to return to the list page

### Actual behaviour
There is no way to return to the list page (query is lost)

### Server configuration
any

**Web server:**
any

**Database:**
mysql

**PHP version:**
any

**phpMyAdmin version:**
4.6.4

### Client configuration
any

**Operating system:**
any

**Comment by Ackshay** [comment]
Hi can you send a screenshot?

**Comment by zoltee** [comment]
[image: Inline image 2]

**Comment by sonanchenko** [comment]
hi2all
do we have any plans to fix/enhance this?

**Comment by robryanx** [comment]
Was looking into this issue as I was using 5.0.2 and had the same issue.
As per the screenshot 5.1.1 shows the query above the export though so I think this can be closed.

![Screenshot from 2021-06-17 19-06-56](https://user-images.githubusercontent.com/310416/122366902-5489d180-cf9f-11eb-8c98-2ade389ca660.png)

**Comment by williamdes** [comment]
> As per the screenshot 5.1.1 shows the query above the export though so I think this can be closed.

I am not sure this was intended, I will have to check
But the query box button does the same, it provides the query

**Comment by williamdes** [comment]
I found out that on 5.0 versions and probably before the message was in the output but buffering was catching it and it would never be printed out.
Now that we improved the code it shows out and should be removed.

**Comment by robryanx** [comment]
Yeah fair enough.
So with that removed I guess the current way to get back to the query would be to copy it from "Show SQL query" and paste it back in the SQL tab?
A refinement on this would be another button next to show SQL query that was essentially "Go back to previous page" option from edit.
I am sure it is much more complicated but the ideal from edit/copy/export on rows would be to have the state work with the browser back button.

Thanks.

**Comment by williamdes** [comment]
Yes exactly this button is exactly for that, I also agree about the back button it would be great. The code is much more organized now and it should be easier to implement
